packages/flutter/test/services/autofill_test.dart

```diff
@@ -0,0 +1,180 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert' show utf8;
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
```

Comment:
Question: Can you also write an integration test in IOS or in Android which uses the real platform channels?

I'm planning to write one for the web but I was curious until then can we also have a test which uses the real platform channels.

Reply:
I feel it won't be easy to set the device up for autofill. On iOS if I remember correctly you need to use an apple account in order to get the personal information (e.g., phone number and street address).